<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Adobe 黑体 Std R</family>
         </font>
        </property>
        <property name="text">
         <string>集数</string>
        </property>
        <property name="indent">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ep_split">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox{font-family:Microsoft YaHei;border:1px;border-color#252525:border-radius:2px;background: #4e4e4e;}
QComboBox QAbstractItemView::item {height:25px;}
QComboBox::drop-down{border:5px;}
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Adobe 黑体 Std R</family>
         </font>
        </property>
        <property name="text">
         <string>场次</string>
        </property>
        <property name="indent">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="seq_split">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox{font-family:Microsoft YaHei;border:1px;border-color#252525:border-radius:2px;background: #4e4e4e;}
QComboBox QAbstractItemView::item {height:25px;}
QComboBox::drop-down{border:5px;}
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Adobe 黑体 Std R</family>
         </font>
        </property>
        <property name="text">
         <string>镜头</string>
        </property>
        <property name="indent">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="shot_split">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox{font-family:Microsoft YaHei;border:1px;border-color#252525:border-radius:2px;background: #4e4e4e;}
QComboBox QAbstractItemView::item {height:25px;}
QComboBox::drop-down{border:5px;}
</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
       </font>
      </property>
      <property name="text">
       <string>上传内容</string>
      </property>
      <property name="indent">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="asset_list">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(30, 30, 30);
border: 0px;</string>
      </property>
      <property name="viewMode">
       <enum>QListView::IconMode</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="description_l">
      <property name="font">
       <font>
        <family>Adobe 黑体 Std R</family>
       </font>
      </property>
      <property name="text">
       <string>上传描述</string>
      </property>
      <property name="indent">
       <number>4</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="description">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(50, 50, 50);
border: 0px;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="shot_step">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox{font-family:Microsoft YaHei;border:1px;border-color#252525:border-radius:2px;background: #4e4e4e;}
QComboBox QAbstractItemView::item {height:25px;}
QComboBox::drop-down{border:5px;}
</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="download_btn">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Adobe 黑体 Std R</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: #4e4e4e;
</string>
        </property>
        <property name="text">
         <string>上传文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
